corehq/blobs/s3db.py

```diff
@@ -34,8 +34,8 @@ def __init__(self, config):
         self.db = boto3.resource(
             's3',
             endpoint_url=config.get("url"),
-            aws_access_key_id=config.get("access_key", ""),
-            aws_secret_access_key=config.get("secret_key", ""),
+            aws_access_key_id=config.get("access_key"),
+            aws_secret_access_key=config.get("secret_key"),
```

Comment:
This works because passing in `""` results in an error, whereas passing in `None` (which is what we'll set it to in `localsettings.py` to use the system default credentials) uses the system default credentials. I don't think you ever actually want a value of `""`, but if you did, you can still pass it in through `localsettings.py`